samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -236,11 +239,36 @@ Test not discoverable advertising by completing `Testing`_ and the following add
 Dependencies
 ************
 
+The sample uses subsystems and firmware components available in the |NCS|.
+For details, see the sections below.
+
+Fast Pair GATT Service
+======================
+
 This sample uses the :ref:`bt_fast_pair_readme` and its dependencies and is configured to meet the requirements of the Fast Pair standard.
 See :ref:`ug_bt_fast_pair` for details about integrating Fast Pair in the |NCS|.
 
 The :ref:`bt_fast_pair_provision_script` is used by the build system to automatically generate the hexadecimal file that contains Fast Pair Model ID and Anti Spoofing Private Key.
 
+Bluetooth LE advertising data providers
```

Comment:
`bt_fast_pair_provision_script` shouldn't be mentioned under `Fast Pair GATT service` header. Please move it elsewhere or rename the header.

Reply:
The script is used "under the hood" by the Fast Pair GATT service, sample does not explicitly use it. Can we leave it as it is?

Reply:
Ok